receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -192,6 +192,11 @@ resource_attributes:
     type: string
     enabled: true
 
+  k8s.container.status.last_terminated_reason:
```

Comment:
I'd definitely be in favor of submitting a PR to get this attribute added to the [semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/k8s.md#container). I am growing more and more worried about how many resource attributes/metrics we have in our k8s receivers that arent specified.

Reply:
I agree. It shouldn't be a blocker, but we need to submit it make sure it's pushed to the sem conv as well